docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -398,3 +398,13 @@ FAQs
         curl -LO "https://dl.k8s.io/release/v1.28.11/bin/linux/amd64/kubectl" && \
         sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl && \
         echo 'export PATH="$PATH:$HOME/.local/bin"' >> ~/.bashrc
+
+Multi-node jobs
+----------------
+
+:ref:`Multi-node jobs <dist-jobs>` are supported on Kubernetes. When a multi-node job is launched, each node in a SkyPilot cluster is provisioned as a separate pod.
```

Comment:
```suggestion
:ref:`Multi-node jobs <dist-jobs>` are supported on Kubernetes. When a multi-node job is launched, each "node" in the SkyPilot cluster is provisioned as a separate pod.
```
Helps clarify that we are not talking about actual "nodes" here as in the Kubernetes sense.